### PR TITLE
Add custom dashboards for data generators

### PR DESCRIPTION
**Pull request by trevorbonas**
*Issue #, if available:*

N/A.

*Description of changes:*

- Custom dashboards have been created for each data generator, to suite their specific data.
- The custom data generator uses the base class dashboard, with a panel for each measure.
- List of data generators has been moved to the "Generate Data" step, so that users have an easier time choosing from among the data generators, as the notebook now has thousands of lines.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by forestmvey — APPROVED** [review]
Looks good from my end. I have validated running the playbook using VSCode and using Sagemaker.

File: integrations/lambda/demo.ipynb [hunk 1]
```diff
@@ -281,6 +281,149 @@
...
+    "        for i, measure_template in enumerate(self.measure_templates):\n",
+    "            measure_name = measure_template['name']\n",
+    "            query = \"\"\n",
+    "            if len(self.measure_templates) > 1:\n",
+    "                dimensions = ', '.join(f\"\\\"{dimension_template['name']}\\\"\" for dimension_template in self.dimension_templates)\n",
+    "                query = f\"SELECT time, \\\"{measure_name}\\\", {dimensions} FROM \\\"{database_name}\\\".\\\"{table_name}\\\" ORDER BY time ASC\"\n",
+    "            elif len(self.measure_templates) == 1:\n",
+    "                query = f\"SELECT * FROM \\\"{database_name}\\\".\\\"{table_name}\\\" ORDER BY time ASC\"\n",
+    "            \n",
+    "            panel = {\n",
```
Comment (sethusrinivasan) [inline integrations/lambda/demo.ipynb:297]: Are there better approaches to build Grafana dashboards programatically? . Generating JSON strings and escaping them is error prone and could be hard to maintain
Examples: https://grafanalib.readthedocs.io/en/stable/getting-started.html

**Review by sethusrinivasan — COMMENTED** [review]
Consider https://grafanalib.readthedocs.io/en/stable/getting-started.html

File: integrations/lambda/demo.ipynb (hunk 1 above)
Comment (trevorbonas) [reply integrations/lambda/demo.ipynb:297]: This isn't a JSON string but a Python dict. Viewing a Jupyter notebook in its raw form causes dicts to appear as escaped JSON strings.

`grafanalib` is in its early stages and doesn't support some panels that I use. Most important of these is the `xychart` panel, which I use to show 2D player movement.

A full list of the panels `grafanalib` supports can be seen [here](https://github.com/weaveworks/grafanalib/blob/5c3b17edaa437f0bc09b5f1b9275dc8fb91689fb/grafanalib/core.py#L77-L103).

Perhaps an alternative is to determine which keys can be left out, to make sure the dicts for the dashboards are small.

Comment (trevorbonas) [reply integrations/lambda/demo.ipynb:297]: I have reduced the number of items in the dashboards dicts, only keeping what is necessary, as many of the dashboards were filled with default values.

**Comment by trevorbonas** [comment]
This PR will be included in a future PR, with most of the Python code moved into its own module.

**Pull request closed without merging**